charts/rancher-backup/Chart.yaml

```diff
@@ -21,6 +21,6 @@ icon: https://charts.rancher.io/assets/logos/backup-restore.svg
 keywords:
 - applications
 - infrastructure
-kubeVersion: '>= 1.16.0-0'
+kubeVersion: '>= 1.16.0-0 < 1.25.0-0'
```

Comment:
We should leave this one as is without the max. This change would just introduce another value we would need to maintain and if we ever miss it, changing the kubeVersion on a released asset is not as easy like we change the kube-version annotation because it breaks stuff. So is better to leave it open ended or not use it at all. @aiyengar2 Please correct me if I'm wrong.

Reply:
😛 The changes in this PR come directly from your review, and this one comes from this comment specifically https://github.com/rancher/charts/pull/1948#discussion_r903121177